src/WorkspaceDependencyViewExtension/WorkspaceDependencyView.xaml.cs

```diff
@@ -455,19 +472,19 @@ public ImageSource Icon
     }
 
     /// <summary>
-    /// Represents information about a local dependency as a row in the dependency table
+    /// Represents information about a dependency as a row in the dependency table
     /// </summary>
-    public class LocalDefinitionRow
+    public class DependencyRow
```

Comment:
I think this is API breaking but should be fine since we did ask folks to not depending on our UI and extension APIs? @reddyashish @mjkkirschner

Reply:
This class was added after 2.12 release. It wouldn't affect users using the daily builds, right?

Reply:
@reddyashish I see, then it should be fine. Thanks!